examples/bootstrap.php

```diff
@@ -26,4 +26,4 @@
 /**
  * Load the credential for the different services
  */
-require_once __DIR__ . '/init.php';
+require_once __DIR__ . '/init.example.php';
```

Comment:
`init.example.php` should be duplicated to `init.php` which is ignored by git. This is where you put your real configuration values.